### PR TITLE
Support rounding for complex types

### PR DESCRIPTION
**Pull request by unnonouno**
`rint` is applied to both real and imag.
related to #1281

### CODE REVIEW

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by kmaehashi** [comment]
Jenkins, test this please.

File: cupy/math/rounding.py [hunk 1]
```diff
@@ -52,4 +52,4 @@
 
     .. seealso:: :func:`numpy.fix`
 
-    ''')
+    ''', support_complex=False)
```
Comment (okuta) [inline cupy/math/rounding.py]: Please revert this because this function is `core.create_ufunc`.

Comment (unnonouno) [reply cupy/math/rounding.py]: i see

**Comment by okuta** [comment]
LGTM!

**Pull request merged**